src/webutil/createRelativeHistory.test.js

```diff
@@ -46,10 +46,13 @@ describe("webutil/createRelativeHistory", () => {
         );
       });
       it("should require a basename", () => {
-        // $ExpectFlowError
-        expect(() => createHistory(undefined, "undefined/")).toThrow(
-          "basename: expected string, got: undefined"
-        );
+        expect(() =>
+          createHistory(
+            // $ExpectFlowError
+            undefined,
+            "undefined/"
+          )
+        ).toThrow("basename: expected string, got: undefined");
```

Comment:
We can't use only ExpectFlowError in this case anymore, because it depended on the suppression propagating to internal function calls too. The implicit function argument would be typed `undefined` and not match the expected `string`. Because that error is no longer suppressed, it should be explicitly typed.